packages/zowe-explorer/src/ZoweExplorerExtender.ts

```diff
@@ -165,12 +163,52 @@ export class ZoweExplorerExtender implements IApiExplorerExtender, IZoweExplorer
             Profiles.getInstance().addToConfigArray(profileTypeConfigurations);
         }
 
+        // Check if schema needs updated when the end user is using a team config
+        if (usingTeamConfig) {
+            this.updateSchema(profileInfo, profileTypeConfigurations);
```

Comment:
is `usingTeamConfig` needed for V3?
I know it's just a port-PR, but wondering if we should just remove the check 😋

Reply:
Ah - yes, this should be removed. Resolved in upcoming commit 34597d6 👍